gtsam/hybrid/DCGaussianMixtureFactor.h

```diff
@@ -43,41 +44,43 @@ class DCGaussianMixtureFactor : public DCFactor {
  public:
   using Base = DCFactor;
   using shared_ptr = boost::shared_ptr<DCGaussianMixtureFactor>;
-  using FactorDecisionTree = DecisionTree<Key, GaussianFactor::shared_ptr>;
+  using Factors = DecisionTree<Key, GaussianFactor::shared_ptr>;
```

Comment:
Not really loving this name change since `Factors` implies a container of factors (aka a factor graph) and not a Decision Tree.

Reply:
It's how factors are stored in this class.